petab/v1/problem.py

```diff
@@ -1104,11 +1104,9 @@ def add_parameter(
         record = {
             PARAMETER_ID: [id_],
         }
-        if estimated is not None:
+        if estimate is not None:
             record[ESTIMATE] = [
-                int(estimated)
-                if isinstance(estimated, bool | int)
-                else estimated
+                int(estimate) if isinstance(estimate, bool | int) else estimate
```

Comment:
```suggestion
                int(estimate) if isinstance(estimate, bool | str) else estimate
```

Reply:
I think probably fine to replace this with simply `[int(estimate)]`

Reply:
Right. None is handled above.

Reply:
Hm, None is not in the signature

Reply:
Ah, yeah, because it's required in the parameters table.
That still an open question how much validation should happen in `Problem.add_*(...)`.

Reply:
I think none (in v2 at least), since the upcoming pydantic object will validate upon construction